ui/src/components/ManageConfigForm/index.tsx

```diff
@@ -143,7 +143,7 @@ export const ManageConfigForm: React.FC<ManageConfigFormProps> = ({
                 <Button
                   className={mixins["ml-2"]}
                   variant="contained"
```

Comment:
I think this would be the secondary action, and therefore outlined.  Also can we make the `Cancel` button a text button as its the tertiary action?

```suggestion
                  variant="outlined"
```